### PR TITLE
feat(update): updating direct purchase constellation doc

### PR DESCRIPTION
**Pull request by Jamie-yu**
Will push after https://github.com/mixer/backend/pull/2034 goes in

### CODE REVIEW

File: src/reference/constellation/events.json [hunk 1]
```diff
@@ -304,8 +304,11 @@
         "description": "Sent when a user uses direct purchase to buy a title on a Streamer's Channel.",
         "payload": {
             "type":"Object",
-            "description":"An empty JSON Object for now"
-        }
+            "description":"A JSON object containing the title of the game sold."
```
Comment (EthanWaite) [inline src/reference/constellation/events.json]: Please could you change the `payload` to include `gameTitle` as a property instead? See this for an example: https://github.com/mixer/developers/blob/df0cf202240bd4064463cae2e7c9c351e06b4a48/src/reference/constellation/events.json#L609

File: src/reference/constellation/events.json [hunk 2]
```diff
@@ -304,8 +304,11 @@
...
+        },
+	"example": {
```
Comment (EthanWaite) [inline src/reference/constellation/events.json]: nit: json formatting

File: src/reference/constellation/events.json (hunk 1 above)
Comment (Jamie-yu) [reply src/reference/constellation/events.json]: Sure. Updated like your example.

File: src/reference/constellation/events.json (hunk 2 above)
Comment (Jamie-yu) [reply src/reference/constellation/events.json]: Fixed the formatting, thanks!

**Pull request merged**